games/m.yaml

```diff
@@ -377,6 +377,25 @@
   updated: 2015-04-15
   url: http://sourceforge.net/projects/mininghaze/
 
+- name: Mininim
+  lang: C
+  framework: Allegro
+  license:
+  - GPL3
+  development: sporadic
+  status: playable
+  content: open
+  originals:
+  - Prince of Persia
+  repo: https://github.com/oitofelix/mininim
+  type: remake
+  updated: 2017-01-12
+  url: http://oitofelix.github.io/mininim/
+  video:
+    youtube:
```

Comment:
only one video supported at the moment

Reply:
Updated it as `youtube: 91QCqPMssj4` is that it?